test/helpers/utils.js

```diff
@@ -13,24 +13,24 @@ window.Utils = {
         if (opts.label) {
             var $text = $el.find('.guide_label').eq(idx);
             expect($text.text()).toBe(opts.label.text);
-            expect(Number($text.attr('x'))).toBeCloseTo(opts.label.x, 0);
-            expect(Number($text.attr('y'))).toBeCloseTo(opts.label.y, 0);
+            expect(Number($text.attr('x'))).toBeCloseTo(opts.label.x, -1);
+            expect(Number($text.attr('y'))).toBeCloseTo(opts.label.y, -1);
```

Comment:
I'm probably just being dense, but how does changing the precision from `0` to `-1` give us a 5 pixel range?